File: app/src/main/java/com/example/entrypay/main_activity.java
package com.example.entrypay;

import android.content.Intent;
import android.os.Bundle;
import android.widget.ImageButton;
import androidx.appcompat.app.AppCompatActivity;

public class main_activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.content_main); // Asegúrate de que este coincide con tu archivo XML

        // Configurar botones
        ImageButton entryButton = findViewById(R.id.entryButton);
        ImageButton payButton = findViewById(R.id.payButton);
        ImageButton intercomButton = findViewById(R.id.intercomButton);

        // Acción para el botón Entry
        entryButton.setOnClickListener(v -> {
            // Lógica para el botón Entry
            Intent entryIntent = new Intent(main_activity.this, EntryActivity.class);
            startActivity(entryIntent);
        });

        // Acción para el botón Pay
        payButton.setOnClickListener(v -> {
            // Lógica para el botón Pay
            Intent payIntent = new Intent(main_activity.this, PayActivity.class);
            startActivity(payIntent);
        });

        // Acción para el botón Intercom
        intercomButton.setOnClickListener(v -> {
            // Lógica para el botón Intercom
            Intent intercomIntent = new Intent(main_activity.this, IntercomActivity.class);
            startActivity(intercomIntent);
        });
    }
}

File: app/src/main/java/com/example/entrypay/MainActivity.kt
package com.example.entrypay

import android.os.Bundle
import android.widget.Button
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import androidx.drawerlayout.widget.DrawerLayout

class MainActivity : AppCompatActivity() {

    private lateinit var drawerLayout: DrawerLayout

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        // Configurar el Toolbar
        val toolbar: Toolbar = findViewById(R.id.toolbar)
        setSupportActionBar(toolbar)

        // Inicializar DrawerLayout
        drawerLayout = findViewById(R.id.drawer_layout)

        // Manejar los clics de los botones del menú
        findViewById<Button>(R.id.menu_entry).setOnClickListener {
            updateContentText("ENTRY seleccionado")
            drawerLayout.close()
        }

        findViewById<Button>(R.id.menu_pago).setOnClickListener {
            updateContentText("PAGO seleccionado")
            drawerLayout.close()
        }

        findViewById<Button>(R.id.menu_intercom).setOnClickListener {
            updateContentText("INTERCOM seleccionado")
            drawerLayout.close()
        }

        findViewById<Button>(R.id.menu_documentos).setOnClickListener {
            updateContentText("MIS DOCUMENTOS seleccionado")
            drawerLayout.close()
        }
    }

    private fun updateContentText(text: String) {
        val contentTextView: TextView = findViewById(R.id.content_text)
        contentTextView.text = text
    }
}
